test/test.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+const test = require('tape');
+window.mapboxgl = require('mapbox-gl');
+window.MapboxAccessibility = require('../src/index').default;
+const polygon = require('./fixtures/polygon');
+
+mapboxgl.accessToken = process.env.MAPBOX_ACCESS_TOKEN;
+
+function createMap(opts) {
+  const options = Object.assign({
+    container: document.createElement('div'),
+    style: 'mapbox://styles/mapbox/streets-v11',
+    center: [-79.387, 43.656],
+    zoom: 13.5
+  }, opts);
+
+  return new mapboxgl.Map(options);
+};
+
+test('MapboxAccessibility: basic', t => {
+  const map = createMap();
+  document.body.appendChild(map.getCanvasContainer());
+
+  map.on('load', () => {
+
+    t.throws(map.addControl(new MapboxAccessibility({
+      accessibleLabelProperty: 'name'
+    })), 'Missing layers array throws');
+
+    t.throws(map.addControl(new MapboxAccessibility({
+      layers: ['poi-label']
+    })), 'Missing accessibleLabelProperty field value throws');
+
+    map.addControl(new MapboxAccessibility({
+      accessibleLabelProperty: 'name',
+      layers: [
+        'poi-label',
+        'transit-label'
+      ]
+    }));
+
+    window.setTimeout(() => {
```

Comment:
This isn't ideal, but it looks like we need to wait for a beat until the accessible elements are on the map.